src/Table.php

```diff
@@ -326,6 +326,18 @@ public function getColumnDecorators($name)
         return is_array($dec) ? $dec : [$dec];
     }
 
+    /**
+     * Return column instance or first instance if using decorator.
+     *
+     * @param $name
+     *
+     * @return mixed
+     */
+    public function getColumn($name)
+    {
+        return is_array($this->columns[$name]) ? $this->columns[$name][0] : $this->columns[$name];
```

Comment:
is `$this->columns[$name]` guaranteed to have always only one element? If no, the decorator API should be unified.

Reply:
No, it is not guaranteed. When adding decorators, the key will not contain the column instance anymore but an array with column instance set at 0 indexes and the rest as decorators. This is enough for fixing this issue right now. We can work on unifying decorator API in a separate PR.

Reply:
Introducing new methods in a hope to fix them later id bad as it takes much more efford to remove them later. If this is only hotfix, make the method private and comment this there - and at best create a PR for it and add that link there too.